pss/pss_test.go

```diff
@@ -1,1660 +1,60 @@
-func TestSendAsym(t *testing.T) {
-	t.Run("32", testSendAsym)
-	t.Run("8", testSendAsym)
-	t.Run("0", testSendAsym)
-}
-
-func testSendAsym(t *testing.T) {
-
-	// address hint size
-	var addrsize int64
-	var err error
-	paramstring := strings.Split(t.Name(), "/")
-	addrsize, _ = strconv.ParseInt(paramstring[1], 10, 0)
-	log.Info("asym send test", "addrsize", addrsize)
-
-	clients, closeSimFunc, err := setupNetwork(2, false)
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer closeSimFunc()
-
-	var topic string
-	err = clients[0].Call(&topic, "pss_stringToTopic", "foo:42")
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	time.Sleep(time.Millisecond * 250)
-
-	var loaddrhex string
-	err = clients[0].Call(&loaddrhex, "pss_baseAddr")
-	if err != nil {
-		t.Fatalf("rpc get node 1 baseaddr fail: %v", err)
-	}
-	loaddrhex = loaddrhex[:2+(addrsize*2)]
-	var roaddrhex string
-	err = clients[1].Call(&roaddrhex, "pss_baseAddr")
-	if err != nil {
-		t.Fatalf("rpc get node 2 baseaddr fail: %v", err)
-	}
-	roaddrhex = roaddrhex[:2+(addrsize*2)]
-
-	// retrieve public key from pss instance
-	// set this public key reciprocally
-	var lpubkey string
-	err = clients[0].Call(&lpubkey, "pss_getPublicKey")
-	if err != nil {
-		t.Fatalf("rpc get node 1 pubkey fail: %v", err)
-	}
-	var rpubkey string
-	err = clients[1].Call(&rpubkey, "pss_getPublicKey")
-	if err != nil {
-		t.Fatalf("rpc get node 2 pubkey fail: %v", err)
-	}
-
-	time.Sleep(time.Millisecond * 500) // replace with hive healthy code
 
-	lmsgC := make(chan APIMsg)
-	lctx, lcancel := context.WithTimeout(context.Background(), time.Second*10)
-	defer lcancel()
-	lsub, err := clients[0].Subscribe(lctx, "pss", lmsgC, "receive", topic, false, false)
-	log.Trace("lsub", "id", lsub)
-	defer lsub.Unsubscribe()
-	rmsgC := make(chan APIMsg)
-	rctx, rcancel := context.WithTimeout(context.Background(), time.Second*10)
-	defer rcancel()
-	rsub, err := clients[1].Subscribe(rctx, "pss", rmsgC, "receive", topic, false, false)
-	log.Trace("rsub", "id", rsub)
-	defer rsub.Unsubscribe()
-
-	// store reciprocal public keys
-	err = clients[0].Call(nil, "pss_setPeerPublicKey", rpubkey, topic, roaddrhex)
+	//Mock the store
+	localStore, err := localstore.New(dir, baseKey, &localstore.Options{})
 	if err != nil {
 		t.Fatal(err)
 	}
-	err = clients[1].Call(nil, "pss_setPeerPublicKey", lpubkey, topic, loaddrhex)
+	var testTargets = [][]byte{
+		{57, 120},
+		{209, 156},
+		{156, 38},
+		{89, 19},
+		{22, 129}}
+	msg := []byte("RECOVERY")
+	//call Send to store trojanChunk in store
+	var ch chunk.Chunk
+	ch, err = Send(ctx, localStore, testTargets, "RECOVERY", msg)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	// send and verify delivery
-	rmsg := []byte("xyzzy")
-	err = clients[0].Call(nil, "pss_sendAsym", rpubkey, topic, hexutil.Encode(rmsg))
+	//verify store, that trojan chunk has been stored correctly
+	var chStored chunk.Chunk
+	chStored, err = localStore.Get(ctx, chunk.ModeGetRequest, ch.Address())
```

Comment:
`chStored` sounds like a `bool` variable; let's try to name variables in a way that it is clear what type of entity we are talking about (use `storedChunk` rather than `chunkStored`)

Reply:
fixed